css-positioning/absolute.css

```diff
@@ -0,0 +1,41 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+  position: absolute;
+  top: 10px;
+  left: 10px;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 10px;
+  left: 240px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: relative;
+  top: 160px;
+}
+
+.overlay {
```

Comment:
please add position: absolute as well as top, bottom, left, and right properties here to make the gradient show up over the image!

Reply:
i tried using top, bottom, left, and right properties, but even with the developer tools the <div .overlay> would show that it has 0x0 pixels so i went with height and width instead